packs/feats/dire-growth.json

```diff
@@ -31,7 +31,86 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.ActorSizeHuge",
+                        "predicate": [
+                            {
+                                "gte": [
+                                    "parent:level",
+                                    4
+                                ]
+                            }
+                        ],
+                        "value": {
+                            "damage": 4,
+                            "reach": 15,
+                            "size": "huge"
+                        }
+                    },
+                    {
+                        "label": "PF2E.ActorSizeLarge",
+                        "predicate": [
+                            {
+                                "or": [
+                                    {
+                                        "lte": [
+                                            "parent:level",
+                                            3
+                                        ]
+                                    },
+                                    {
+                                        "gte": [
+                                            "parent:level",
+                                            6
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": {
+                            "damage": 2,
+                            "reach": 10,
+                            "size": "large"
+                        }
+                    }
+                ],
+                "flag": "enlarge",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.CreatureSize"
+            },
```

Comment:
The Choice Set isn't needed. You are under the effects of enlarge at its base rank.

Reply:
Changes made - check when you get a moment.

Thanks @pedrogrullada :)